controllers/topics/index.js

```diff
@@ -285,6 +285,19 @@ const getFollowerList = async (req, res) => {
   try {
     const searchQuery = search ? `AND LOWER(users.username) LIKE LOWER(:search)` : '';
 
+    let user_topics = await sequelize.query(
+      `select tp.topic_id from topics as tp
+      left join user_topics as utp on tp.topic_id = utp.topic_id
+      where utp.user_id = :userId`,
+      {
+        type: QueryTypes.SELECT,
+        replacements: {
+          userId
+        }
+      }
+    );
+    let topicIds = user_topics.map((topic) => topic.topic_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The query to fetch user topics is not protected against SQL injection. It's a good practice to use parameterized queries or ORM methods that automatically escape input to prevent SQL injection attacks.
```diff
-    let user_topics = await sequelize.query(
-      `select tp.topic_id from topics as tp
-      left join user_topics as utp on tp.topic_id = utp.topic_id
-      where utp.user_id = :userId`,
-      {
-        type: QueryTypes.SELECT,
-        replacements: {
-          userId
-        }
-      }
-    );
+    let user_topics = await UserTopic.findAll({
+      attributes: ['topic_id'],
+      where: { user_id: userId },
+      include: [{
+        model: Topic,
+        as: 'topics',
+        attributes: ['topic_id']
+      }]
+    });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->